drivers/i2c/i2c_sc18is600.c

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (c) 2017-2018 Antmicro Ltd <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <misc/printk.h>
+#include <i2c.h>
+#include <spi.h>
```

Comment:
Note that there will be very soon SPI API changes: see PR #5921 
It not big changes but you'll have to adapt your driver to it.